Added welcome video to the home page banner

jamiewaese-uncharted opened the pull request:
BEFORE
![image](https://github.com/DARPA-ASKEM/terarium/assets/120480244/95c96f1d-1f4b-417c-a248-d56197b2d95c)


AFTER
![image](https://github.com/DARPA-ASKEM/terarium/assets/120480244/14686b92-3a9a-4926-8e11-9abda7b4410b)



Note, this is the video we made for the CHI submission. We can easily change videos when we have a new one.

mwdchang reviewed (changes requested):
Please do not commit larger files (say larger than 4MB) into the git repo. This makes things rather difficult to work with afterwards as it comes a part of the permanent history and makes things really slow with working with git.

We should put the video somewhere else that is more dedicated to housing multimedia....like maybe on a publicly available S3 bucket or in the documentation-image or something.

jamiewaese-uncharted commented:
I removed the video file and updated the thumbnail image. Should be good to go... ?
![image](https://github.com/DARPA-ASKEM/terarium/assets/120480244/7e24080a-0090-494b-938d-9673c7cc29d1)
